conftest.py

```
@@ -0,0 +1,27 @@
+def sample_config():
+    return {
+        'num_features': ['num1', 'num2'],
+        'cat_features': ['cat1', 'cat2'],
+        'target': 'target'
+    }
+
+@pytest.fixture
+def data_processor(tmp_path, sample_data, sample_config):
+    csv_path = tmp_path / "test_data.csv"
+    sample_data.to_csv(csv_path, index=False)
+    return DataProcessor(csv_path, sample_config)
```

Ruff

No newline at end of file

Add trailing newline

(W292)